app/logic/stake_logic.rb

```diff
@@ -152,8 +152,11 @@ def update_validator_stats
         uptimes = []
         lifetimes = []
         scores = []
+        total_active_stake = 0
 
-        Validator.where(id: validator_ids).each do |validator|
+        Validator.where(id: validator_ids).includes(:stake_accounts).each do |validator|
+          val_active_stake = validator.stake_accounts.active.where(stake_pool: pool).pluck(:active_stake).sum
```

Comment:
`val_active_stake` is a shortcut of `validator_active_stake` ? Let's use the longer version.